docs/docusaurus/docs/core/set_up_a_gx_environment/_install_gx/_emr_spark_installation.md

```diff
@@ -0,0 +1,26 @@
+Use the information provided here to install GX on an EMR Spark cluster and instantiate a Data Context without a full configuration directory.
+
+## Additional prerequisites
+
+- An EMR Spark cluster.
+- Access to the EMR Spark notebook.
+
+## Installation and setup
+
+1. To install Great Expectations on your EMR Spark cluster copy this code snippet into a cell in your EMR Spark notebook and then run it:
```

Comment:
```suggestion
1. To install GX on your EMR Spark cluster copy this code snippet into a cell in your EMR Spark notebook and then run it:
```
Use _GX_ or _Great Expectations_ consistently. Use find and replace to correct inconsistencies.